### DISCUSSION TITLE
Disable transition effect

### DISCUSSION THREAD

**Discussion by paulettepaillette** [google/model-viewer#3340] [Q&A] [answered]
### Description

Please provide a detailed description of the bug, how to reproduce it, and the
expected behavior.  Always include a code snippet, screenshot, any errors
reported in the console, and the model to help us understand and fix the
problem.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

https://dev.bklt.fr/test/slider.html 

### Version

- model-viewer: vX.X.X
- https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js

### Browser Affected

- [X] Chrome, version: 100.0.4896.60
- [ ] Edge
- [X] Firefox, version: 98.0.2 
- [ ] IE
- [X] Safari, version: 15.0 

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [X] MacOS : Catalina, 10.15.7
- [ ] Windows

### AR

- [X] WebXR
- [ ] SceneViewer
- [ ] QuickLook

Hi, 

I would like to know if it's possible to disable the transition effect between the poster and the 3d model viewer and when I click on the second slider. I don't know if I'm really clear, but when the slide change, at the beginning, I have an opacity transition, the background is clearer. I have the same effect when the first slide appears after the poster image. I would like to disable this effect because it's weird, it creates a demarcation with the rest of my div. 

Thanks

**Comment by elalish** [comment, marked as answer]
I believe this is what you want: https://modelviewer.dev/docs/#entrydocs-loading-parts-defaultProgressMask

**Comment by paulettepaillette** [comment]
Yes ! Thanks you :)
